samples/inputs/badge/outlined/App.razor

```diff
@@ -1,9 +1,13 @@
 @using IgniteUI.Blazor.Controls
 @inject IIgniteUIBlazor IgniteUIBlazor
 
-<div class="container vertical">
-    <IgbBadge Outlined="true"/>
-</div>
+<body>
+    <div id="root">
+        <div class="sample center">
+            <IgbBadge Outlined="true" />
+        </div>
+    </div>
+</body>
 
```

Comment:
should never use, `<body>` or `<div id="root">` in .razor files.
You should check if wwwroot/index.html has `<body>` and `<app class="app">` e.g.
https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/category-chart/annotations/wwwroot/index.html#L20